src/SourceLocator/Exception/InvalidDirectory.php

```diff
@@ -23,7 +23,7 @@ public static function fromNonDirectory($nonDirectory)
      *
      * @return InvalidDirectory
      */
-    public static function fromNonStringValue($nonStringValue)
+    public static function fromNonStringValue($nonStringValue) : self
```

Comment:
Nope, it's used in `DirectoriesSourceLocator` when iterating the `array` of `$directories` given, so still used